src/ReactiveMarbles.MarbleLocator.Tests/ReactiveMarbles.MarbleLocator.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
```

Comment:
Might be worth doing a Update on the dependencies of this one. Just be careful that none of the ones in the Directory.Build.Props change.